apps/server/src/modules/tool/uc/lti11.uc.ts

```diff
@@ -19,6 +19,18 @@ export class Lti11Uc {
 		private readonly userService: UserService
 	) {}
 
+	async getLaunchParameters(currentUser: ICurrentUser, toolId: string, courseId: string): Promise<Authorization> {
```

Comment:
Please do not pass the `ICurrentUser` as a parameter to the use case. Use the `userId` instead.

Reply:
This is already changed in `N21-410-ctl-delete`